vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/IntegerField.java

```diff
@@ -26,6 +26,7 @@
  *
  * @author Vaadin Ltd.
  */
+@SuppressWarnings("deprecation")
```

Comment:
Suppression is not needed here as this class does not use any deprecated symbols.

Reply:
Addressed in the latest commit.